src/main/java/tsunami/security/scanner/utilities/App.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
```

Comment:
Can you change the file header comments to the copyright header? You can find it on the home page of this repo.